src/prpy/tsr/tsr.py

```diff
@@ -51,15 +51,20 @@ def __init__(self, T0_w=None, Tw_e=None, Bw=None,
         pibound = (abs(Bw[3:6, :]) < numpy.pi + EPSILON)
         if pibound.any() is False:
             raise(ValueError('Rotations must be [-pi, pi]', pibound))
-        # We will now deliberately exceed the bound to maintain both:
-        # 1. Bw[i,1] > Bw[i,0] which is necessary for LBFGS-B
-        # 2. signed rotations, necessary for expressiveness
-        for rot_idx in range(3, 6):
-            if Bw[rot_idx, 0] > Bw[rot_idx, 1] + EPSILON:
-                Bw[rot_idx, 1] += 2*pi
+        
         self.T0_w = T0_w
         self.Tw_e = Tw_e
         self.Bw = Bw
+        
+        # We will now create a continuous version of the bound to maintain:
+        # 1. Bw[i,1] > Bw[i,0] which is necessary for LBFGS-B
+        # 2. signed rotations, necessary for expressiveness
```

Comment:
This might be an outdated comment.  The "signed rotations" really mean being able to explicitly specify inner and outer angular bounds, which we are proposing to use reversed Bw ordering for.

The intuition is that the Bw[:,0] indicates a minimum angle, and then _increases_ from that to the Bw[:,1] value, regardless of angle wrapping.